subsys/bluetooth/shell/gatt.c

```diff
@@ -950,6 +950,70 @@ static int cmd_notify(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+#if defined(CONFIG_BT_GATT_NOTIFY_MULTIPLE)
+static int cmd_notify_mult(const struct shell *sh, size_t argc, char *argv[])
+{
+	const size_t max_cnt = CONFIG_BT_L2CAP_TX_BUF_COUNT;
+	struct bt_gatt_notify_params params[max_cnt];
+	const size_t min_cnt = 1U;
+	unsigned long data;
+	unsigned long cnt;
+	uint16_t cnt_u16;
+	int err = 0;
+
+	if (!echo_enabled) {
+		shell_error(sh, "Nofication not enabled");
```

Comment:
I have a suggestion for UX improvement:
```suggestion
		shell_error(sh, "No clients have enabled notifications for this attribute.");
```

Reply:
Changed it to `"No clients have enabled notifications for the vnd1 CCC."` as it specifically checks for the CCC of the `vnd1_echo` attribute.